rust/gui-client/src-tauri/deb_files/firezone-client-ipc.service

```diff
@@ -7,8 +7,8 @@ CapabilityBoundingSet=CAP_CHOWN CAP_NET_ADMIN
 DeviceAllow=/dev/net/tun
 LockPersonality=true
 LogsDirectory=dev.firezone.client
-# Allow users in `firezone` group to delete log files
-LogsDirectoryMode=775
+# Allow anyone to read log files
+LogsDirectoryMode=755
```

Comment:
Just out of curiosity, why not `644`? Why do we need the execute bit set?

Reply:
Oops

Reply:
Oh, it's the mode of the directory itself. `Defaults to 0755` per systemd.exec docs.
I should double-check that the logs themselves are inheriting this, cause I might have mixed something up here.